smoke-tests-otel-starter/src/main/java/io/opentelemetry/spring/smoketest/OtelSpringStarterSmokeTestApplication.java

```diff
@@ -18,4 +23,16 @@ public OtelSpringStarterSmokeTestApplication() {}
   public static void main(String[] args) {
     SpringApplication.run(OtelSpringStarterSmokeTestApplication.class);
   }
+
+  @Bean
+  AutoConfigurationCustomizerProvider propagatorCustomizer() {
```

Comment:
I would have declared this bean in the `OtelSpringStarterSmokeTest` class to have this custom configuration closed to the assertion checking it.